### DISCUSSION TITLE
Foxess T-series

### DISCUSSION THREAD

**Discussion by nasx69** [nathanmarlor/foxess_modbus#323] [Q&A] [answered]
As the T-series are not supported today, will it be supported in the future updates? 

Or at least I am not able to configure my T-series with the USB to Modbus adapter. 

BR
Nicklas

**Comment by canton7** [comment, marked as answer]
Please see #266

**Reply by nasx69** [reply]
Thanks! I tried to search but obviously did not try in the correct way!

**Reply by canton7** [reply]
No worries!
